.circleci/config.yml

```diff
@@ -1,3 +1,10 @@
+no-backports: &no-backports
+  name: Skip any branches called backport* but not PRs
+  command: |
+    if [[ -z "${CIRCLE_PR_NUMBER}" && "${CIRCLE_BRANCH}" = *"backport"* ]]; then
```

Comment:
My logic is either broken or `CIRCLE_PR_NUMBER` is blank for a PR unless its a forked PR.